nebula_ros/launch/continental_launch_all_hw.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<launch>
+    <arg name="sensor_model" default="ARS548"/>
+    <arg name="frame_id"  default="continental_ars548"/>
```

Comment:
`frame_id` is declared as `continental` in `nebula_ros/src/continental/continental_ars548_hw_interface_ros_wrapper.cpp`, please make them the same everywhere.
Personally, I think changing it to just `continental` is more readable.

Reply:
Thanks !
Keeping up with nebula conventions, I changed them to continental in
https://github.com/tier4/nebula/commit/182c57162b7234049fd62e73bdc1079a8c5c8dd6